README.md

```diff
@@ -1,10 +1,10 @@
-# Survey of Domain Verification Techniques using DNS
+# Domain Verification Techniques using DNS
 
-This is the working area for the individual Internet-Draft, "Survey of Domain Verification Techniques using DNS".
+This is the working area for the individual Internet-Draft, "Domain Verification Techniques using DNS".
 
-* [Editor's Copy](https://ShivanKaul.github.io/draft-sahib-domain-verification-techniques/#go.draft-sahib-domain-verification-techniques-latest.html)
-* [Individual Draft](https://tools.ietf.org/html/draft-sahib-domain-verification-techniques-latest)
-* [Compare Editor's Copy to Individual Draft](https://ShivanKaul.github.io/draft-sahib-domain-verification-techniques/#go.draft-sahib-domain-verification-techniques-latest.diff)
+* [Editor's Copy](https://ShivanKaul.github.io/draft-ietf-dnsop-domain-verification-techniques/#go.draft-ietf-dnsop-domain-verification-techniques-latest.html)
```

Comment:
this is still pointing to ShivanKaul.github.io instead of https://github.com/ietf-wg-dnsop/draft-ietf-dnsop-domain-verification-techniques/

Reply:
Think I'll just remove the README changes, will do that in a separate PR, needs some more tooling fixes to make `gh-pages` work correctly.